core/src/main/java/bio/terra/pearl/core/service/kit/PepperKitStatus.java

```diff
@@ -12,9 +13,15 @@
 public class PepperKitStatus {
     private String juniperKitId;
     private String currentStatus;
+    private String dsmShippingLabel;
+    @JsonProperty("participantId")
```

Comment:
this could use a comment explaining why we want to export this as participantId -- I suspect because it's just giving us a Participant.id that we gave to it?  But I thought DSM was always operating on our Enrollee shortcodes (or ids).  Should DSM know that two kit requests for different enrollees in different studies actually correspond to the same Juniper participant?

Reply:
This is a great question because it highlights the need for some documentation in these files. I'm in the process of adding that.

> why we want to export this as participantId

It's the other way around. In the JSON we receive from DSM, this is called `participantId`. Not knowing what to expect the value to be, I named the property `dsmParticipantId`. However, I can now see that this is echoing the enrollee ID passed to DSM via `PepperDSMKitRequest.juniperParticipantId`. Therefore, removing the `dsm` prefix (and therefore the `@JsonProperty` annotation) makes sense.

Alternatively, we could keep `@JsonProperty("participantId")` and rename the field to `enrolleeShortcode` (along with a similar change in `PepperDSMKitRequest`) which would make these files the definitive place where differences in vocabulary between Juniper and Pepper are defined. In fact, I think I like that the best. @devonbush, what do you think?

I think your remaining questions are explained by that difference in vocabulary; DSM is saying "participantId" while Juniper is using enrollee shortcode.